graphqlapi/src/main/java/io/stargate/graphql/web/resources/schemafirst/SchemaFirstCache.java

```diff
@@ -74,12 +76,13 @@ public GraphQL getAdminGraphql() {
     return adminGraphql;
   }
 
-  public GraphQL getGraphql(String namespace, Map<String, String> headers) throws Exception {
-    return getGraphql(persistence.decorateKeyspaceName(namespace, headers));
+  public GraphQL getGraphql(String namespace, Map<String, String> headers, Optional<UUID> version)
+      throws Exception {
+    return getGraphql(persistence.decorateKeyspaceName(namespace, headers), version);
   }
 
-  private GraphQL getGraphql(String namespace) throws Exception {
-    SchemaSource latestSource = schemaSourceDao.getLatest(namespace);
+  private GraphQL getGraphql(String namespace, Optional<UUID> version) throws Exception {
```

Comment:
I am not sure if we should modify this method to get the schema for a specific version or create a new method.
The reason for this concern is that this method uses the `graphqlHolders` map, which keeps only the last holder per namespace.
If customers will often request schema with non-lastest version, this "cache" will have a lot of miss (because elements will be constantly replaced), wdyt?